dev/packages/beats/coredns-0.0.1/dataset/log/agent/stream/log.yml.hbs

```diff
@@ -0,0 +1,7 @@
+type: log
```

Comment:
I think this should be `input: log` as based on this the mapping to the input is done? Or did this become obsolete because of the path?

Reply:
I think you're right... I will adjust the PR.

Reply:
Fixed!